modules/nw-ovn-kubernetes-migration-about.adoc

```diff
@@ -60,6 +60,14 @@ While the OVN-Kubernetes network plugin implements many of the capabilities pres
 
 The following sections highlight the differences in configuration between the aforementioned capabilities in OVN-Kubernetes and OpenShift SDN network plugins.
 
+[discrete]
+[id="migrating-sdn-primary-interface_{context}"]
+=== Primary network interface
+
+The OpenShift SDN plugin defines the primary network interface in a `NodeNetworkConfigurationPolicy` (NNCP) custom resource (CR) while the OVN-Kubernetes network plugin defines the interface in a `MachineConfig` object. If you try to migrate the NNCP to the OVN-Kubernetes network plugin, the `nmstate` API fails to apply the NNCP configuration to your {product-title} cluster.
```

Comment:
This isn't exactly right. SDN _allows_ you to apply an NNCP to the primary interface, but that isn't always the case. I would simplify this to say something like:

"The OpenShift SDN plugin allows application of an NNCP to the primary interface on a node, while the OVNKubernetes plugin does not. If you have an NNCP applied to the primary interface, you must delete it before migrating. Deleting the NNCP will not remove the configuration, but it will no longer be possible to manage it with Kubernetes-NMState because the configure-ovs.sh shell script will now manage that interface."

Reply:
:heavy_check_mark: